app/models/revision.rb

```diff
@@ -17,16 +17,25 @@
 #  wp10_previous  :float(24)
 #  ithenticate_id :integer
 #  report_url     :string(255)
+#  wiki_id        :integer
+#  native_id      :integer
+#  page_id        :integer
```

Comment:
If I understand correctly, `page_id` (soon to be `mw_page_id`, I think?) will the the "new" multiwiki-aware version of `article_id`? If so, maybe a comment would be helpful to explain this? (I stared down the MW DB schema for a few, attempting to grok... ;p )